<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlbumAddDialog</class>
 <widget class="QDialog" name="AlbumAddDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>175</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add album</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Album Page</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="ownerId_label">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Owner Id</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QRadioButton" name="asId_radioButton">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QSpinBox" name="ownerId_spinBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="minimum">
          <number>-999999999</number>
         </property>
         <property name="maximum">
          <number>999999999</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="albumId_label">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Album Id</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="albumId_spinBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>999999999</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="3">
      <widget class="QToolButton" name="downloadDirectory_toolButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="downloadDirectory_label">
       <property name="text">
        <string>Download to</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="asLink_radioButton">
       <property name="text">
        <string/>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="downloadDirectory_lineEdit"/>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="link_lineEdit"/>
     </item>
     <item row="4" column="2">
      <widget class="QCheckBox" name="checkBox_autoStart">
       <property name="text">
        <string>Start download after adding</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>asLink_radioButton</tabstop>
  <tabstop>link_lineEdit</tabstop>
  <tabstop>asId_radioButton</tabstop>
  <tabstop>ownerId_spinBox</tabstop>
  <tabstop>albumId_spinBox</tabstop>
  <tabstop>downloadDirectory_lineEdit</tabstop>
  <tabstop>downloadDirectory_toolButton</tabstop>
  <tabstop>checkBox_autoStart</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AlbumAddDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AlbumAddDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>asLink_radioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_3</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>22</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>82</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>asLink_radioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>link_lineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>22</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>asId_radioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>ownerId_label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>22</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>97</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>asId_radioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>ownerId_spinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>22</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>asId_radioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>albumId_label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>22</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>asId_radioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>albumId_spinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>22</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>463</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
